java/jpf-regression/ExSymExeFNEG_true/Main.java

```diff
@@ -31,7 +31,8 @@
   public static void main(String[] args) {
     float x = Verifier.nondetFloat();
     Main inst = new Main();
-    inst.test(x);
+    if (x>=0 && !Float.isNaN(x))
```

Comment:
Why is NaN a problem? A comparison with NaN is always false.
The conditional `y > 0`  would be false and the else branch is taken.

Reply:
Fixed.